code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -323,7 +323,7 @@
 			module_sprites["Marina-PK"] = "marinaSC"
 			module_sprites["#9"] = "servbot-sec"
 			module_sprites["Kodiak"] = "kodiak-sec"
-			to_chat(src, "<span class='warning'><big><b>You are expected to follow orders and prevent harm above all else. Space law means nothing to you- you are not a Security Officer.</b></big></span>")
+			to_chat(src, "<span class='warning'><big><b>You are expected to follow your laws above all else. Space law means nothing to you- you are not a Security Officer.</b></big></span>")
```

Comment:
You're still saying 1.  A mouthful and 2.  That space law doesn't apply when it does under robocop.